boards/arm/nrf52840_pca10059/board.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016-2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/* Push button switch 0
```

Comment:
These could be defined in DTS, instead

Reply:
@ioannisg , sure these can be moved to the DTS but I'd rather
do that in a separate PR so that we can get this one in sooner than later.

Reply:
Well, in the nrf52810_pca10040, we did it the board support PR, so we skipped the need of a follow-up PR. Take a look how to do it, if you need guidelines: https://github.com/zephyrproject-rtos/zephyr/pull/7915. I recommend to do it here.

Reply:
Agreed, let's do it here

Reply:
@carlescufi @ioannisg I have made this change.